plugins/dash/packet-dash.c

```diff
@@ -229,13 +240,22 @@ static const value_string spork_description[] =
   { 10001, "SPORK_2_INSTANTSEND_ENABLED" },
   { 10002, "SPORK_3_INSTANTSEND_BLOCK_FILTERING" },
   { 10004, "SPORK_5_INSTANTSEND_MAX_VALUE" },
-  { 10007, "SPORK_8_MASTERNODE_PAYMENT_ENFORCEMENT" },
```

Comment:
Going to retain the deprecated sporks since these messages are still relayed around the network.